### PR TITLE
Relative wind

### PR DESCRIPTION
**Pull request by AntoineBernigaud**
Changed hycom_opt and mom_momtum.F to include a parameter "relative_wnd_coef" that multiplies usur and vsur to take into account the relative wind speed to compute the momentum. 

Parameter added to the hycom_opt namelist file.

### CODE REVIEW

File: TP5a0.06/expt_01.0/hycom_opt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 &hycom_nml
-    write_arche= .false.
+    write_arche= .false.,
+    relative_wnd_coef = 0.7 ! coefficient of the surface wind for momentum
```
Comment (lbertino) [inline TP5a0.06/expt_01.0/hycom_opt:3]: Suggested comment: 
! Coupling coefficient in Renault et al. JPO 2017 https://doi.org/10.1175/JPO-D-16-0168.1
! 0.0 is absolute winds, 1.0 is relative winds.

File: hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/Makefile [hunk 2]
```diff
@@ -136,13 +138,13 @@ trcupd.o:  trcupd.F  mod_xc.o mod_cb_arrays.o                     mod_pipe.o
 wtime.o:   wtime.F
 mod_hycom.o: \
         mod_hycom.F  mod_xc.o mod_cb_arrays.o            mod_za.o mod_pipe.o \
+								  mod_NERSCnml.o \
 	                                                          mod_incupd.o \
 	                                                          mod_mean.o \
 	                                                          mod_floats.o \
 	                                                          mod_momtum.o \
 	                                                          mod_tsadvc.o \
 	                                                          mod_stokes.o \
```
Comment (TillRasmussen) [inline hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/Makefile:147]: The \ should in the end of mod_stokes.o should be removed

File: hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_momtum.F [hunk 3]
```diff
@@ -43,6 +43,7 @@ subroutine momtum_hs(m,n)
 #ifdef CPL_OASIS_HYCOM
       use mod_cpl_oasis_init
 #endif
+      use mod_NERSCnml, only : relative_wnd_coef, nersc_init
```
Comment (TillRasmussen) [inline hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_momtum.F:46]: nersc_init is not used here. No need to include it. It is read in just after blkdat.

**Review by TillRasmussen — COMMENTED** [review]
I left a few comments in the code. It looks correct but I have not compiled nor run the code.
Did you check that you got the same result as before when relative_wnd_coef = 1.0?

**Review by TillRasmussen — COMMENTED** [review]
I think that it is ok now. One thing to consider is that this is a relatively small add on to the code that includes many commits. This is a more general thing but maybe it is more obvious to follow the history in git if some of these were "sqaushed". See e.g. https://www.git-tower.com/learn/git/faq/git-squash. This is a more general thing thus what is your thoughts  @lbertino @lilleannette @Jipingnersc (more people could probably be added to the list)

**Comment by AntoineBernigaud** [comment]
I tested again if I got the same results as before when relative_wnd_coef = 1.0, but I got some differences. I am trying to understand them.

**Comment by TillRasmussen** [comment]
Is it round off or more significant?

**Comment by lilleannette** [comment]
Hi Till,

‘Squashing’ when merging seems like a good idea to me: I see there are two options, rebasing the branch before merging or squashing when merging.  I did the second one when merging some changes yesterday, buy I don’t know git well enough to know if one is better than the other.

Annette


Annette Samuelsen
forskningsleder | Research Leader

***@***.*** | +47 932 38 202


Stiftelsen Nansen Senter for Miljø og Fjernmåling
Nansen Environmental and Remote Sensing Center
Jahnebakken 3 | N-5007 Bergen | Norway
***@***.*** | nersc.no<https://nersc.no> | +47 55 20 58 00

[cid:logo_de1c6baf-5da6-4f53-b1e5-34827df1144c.jpg]


Konfidensialitet. Informasjonen i denne e-posten er utelukkende ment for bruk av adressaten(e),
og kan inneholde konfidensiell informasjon som ikke kan deles uten avsenders skriftlige samtykke.

Notice of confidentiality. The information contained in this e-mail is intended solely for the use of
the addressee(s) and may contain confidential or legally privileged information.
From: TRasmussen ***@***.***>
Date: Monday, 11 September 2023 at 10:41
To: nansencenter/NERSC-HYCOM-CICE ***@***.***>
Cc: Annette Samuelsen ***@***.***>, Mention ***@***.***>
Subject: Re: [nansencenter/NERSC-HYCOM-CICE] Relative wind (PR #190)

@TillRasmussen commented on this pull request.

I think that it is ok now. One thing to consider is that this is a relatively small add on to the code that includes many commits. This is a more general thing but maybe it is more obvious to follow the history in git if some of these were "sqaushed". See e.g. https://www.git-tower.com/learn/git/faq/git-squash. This is a more general thing thus what is your thoughts @lbertino<https://github.com/lbertino> @lilleannette<https://github.com/lilleannette> @Jipingnersc<https://github.com/Jipingnersc> (more people could probably be added to the list)

**Comment by AntoineBernigaud** [comment]
I am closing the pull request to make a new one clearer with merged commits

**Pull request closed without merging**